samples/testbot/deployment-service/DeployToFunctionApp/run.ps1

```diff
@@ -0,0 +1,90 @@
+$ErrorActionPreference = "Stop"
+
+# Move out of the current directory or the handle will be held on it
+Set-Location $env:TEMP
+
+# Create the zip file for publishing
+$targetZipFilePath = "$env:TEMP\app.zip"
+if (Test-Path -Path $targetZipFilePath) {
+    Write-Host "Deleting existing $targetZipFilePath..."
+    Remove-Item $targetZipFilePath -ErrorAction Ignore -Force
+}
+
+$zipSrc = "$buildDir\bin\Debug\$Framework\publish\"
+Write-Host "Zipping $zipSrc into $targetZipFilePath..."
+[System.IO.Compression.ZipFile]::CreateFromDirectory($zipSrc, $targetZipFilePath)
+Write-Host "$targetZipFilePath created successfully!"
+
+#"**********DEPLOY - Connecting to Azure Account***********"
+$azurePassword = ConvertTo-SecureString $env:DFTEST_AAD_CLIENT_SECRET -AsPlainText -Force
+$psCred = New-Object System.Management.Automation.PSCredential($env:AZURE_APP_ID , $azurePassword)
+Connect-AzAccount -Credential $psCred -Tenant $env:AZURE_TENANT_ID -ServicePrincipal
+
+$subscriptionId = $Request.Body.subscriptionId
+Write-Host "sub id - $subscriptionId"
+Set-AzContext -SubscriptionId $subscriptionId
+
+#"**********Deploying a zip file to Function***********"
+$appName = $Request.Body.appName
+$resourceGroup = $Request.Body.resourceGroup ?? "perf-testing"
+Write-Host "Publishing $targetZipFilePath to $appName in resource group $resourceGroup..."
+Write-Host "Publish-AzWebApp -ResourceGroupName $resourceGroup -Name $appName -DefaultProfile $defaultProfile -ArchivePath $targetZipFilePath -Force"
+Publish-AzWebApp -ResourceGroupName $resourceGroup -Name $appName -DefaultProfile $defaultProfile -ArchivePath $targetZipFilePath -Force
+
+# Triggering the many instances function
```

Comment:
What is the "many instances" function?

Reply:
That comment is outdated, but here's the link to the ManySequences test: https://github.com/Azure/azure-functions-durable-extension/blob/dev/test/DFPerfScenarios/Tests/ManySequencesTest.cs